config/v2/exithandler/webhook/kustomization.yaml

```diff
@@ -5,7 +5,6 @@ resources:
 - mutatingwebhookconfig.yaml
 - role.yaml
 - rolebinding.yaml
-- secret.yaml
```

Comment:
@amadhusu the resources here are part of the exithandler/webhook component, this is how the organization is for the `v2` directory and we should remain consistent. 

Can you make the following changes: 

* move this secret back to `config/v2/exithandler/webhook/secret.yaml` 
* same with `config/v2/kfptask/webhook/secret.yaml` 
* move `config/v2/secrets/tektonpipelineloopwebhookcertssecret.yaml` to `config/v2/pipelineloop/webhook/secret.yaml`
* rename `config/v2/configmaps` to `config/v2/tektonconfigs`

Reply:
@HumairAK - To clarify, you want me to revert the changes to the locations of these secrets but include these secrets directly via the base `kustomization.yaml` (https://github.com/opendatahub-io/data-science-pipelines-operator/tree/dspv2/config/overlays/make-v2deploy)?

OR

do you want me to revert the changes to the location of these secrets and also the revert the `kustomization.yaml` in each webhook subdirectory which would circle us back to the original issue of `namePrefix` which would result in the bullet points mentioned below?

- `config/v2/exithandler/webhook/secret.yaml` would become `data-science-pipeline-operator-kfp-exithandler-webhook-certs` after deployment to OCP instead of `kfp-exithandler-webhook-certs`
- `config/v2/kfptask/webhook/secret.yaml` would become `data-science-pipeline-operator-kfptask-webhook-certs` after deployment to OCP instead of `kfptask-webhook-certs`
- config/v2/pipelineloop/webhook/secret.yaml would become `data-science-pipeline-operator-tektonpipelineloop-webhook-certs` instead of `tektonpipelineloop-webhook-certs`